seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -330,6 +333,17 @@ def test_saved_variant_data(self):
         self.assertListEqual(variants['SV0000002_1248367227_r0390_100']['familyGuids'], ['F000002_2'])
         self.assertEqual(set(response_json['familiesByGuid'].keys()), {'F000001_1', 'F000002_2', 'F000012_12'})
 
+        # Test empty project
```

Comment:
```suggestion
        # Test project with no saved variants
```

Reply:
this feels silly but if it's true it just makes it really clear what empty means and what this block of code is really testing

Reply:
The name of the project is empty project and this is what its called in several other places, so I am going to leave it as-is for consistency with all the other test files